src/main/java/marquez/db/DatasetDao.java

```diff
@@ -35,23 +35,23 @@
 @RegisterRowMapper(DatasetRowMapper.class)
 public interface DatasetDao {
   @CreateSqlObject
-  DataSourceDao createDataSourceDao();
+  DatasourceDao createDatasourceDao();
 
   @CreateSqlObject
   DbTableVersionDao createDbTableVersionDao();
 
   @SqlUpdate(
       "INSERT INTO datasets (guid, namespace_guid, datasource_uuid, urn, description) "
-          + "VALUES (:uuid, :namespaceUuid, :dataSourceUuid, :urn, :description)")
+          + "VALUES (:uuid, :namespaceUuid, :datasourceUuid, :urn, :description)")
```

Comment:
Did we also apply the column rename in our DB schema?

Reply:
This doesn't affect the DB, where the field name is `datasource_uuid`:
`ALTER TABLE datasets ALTER COLUMN datasource_uuid SET NOT NULL;`

This is just the name of the field from `DatasetRow`, which is also in that same casing:
`@Getter @NonNull private final UUID datasourceUuid;`